### ISSUE TITLE
Plugin not working on Manjaro

### ISSUE THREAD

**Issue by micmalti** [alex1701c/krunner-vscodeprojects#15]
Hi, first of all thank you for developing this plugin. 

I managed to successfully install the AUR version 2.0.0-1 on Manjaro with KDE Plasma 6.0.5, yet the plugin is seemingly not being activated by krunner, so I'm unable to reproduce the same behaviour as shown in the README.

Am I perhaps missing some additional setup on the VS Code side?

**Comment by alex1701c** [comment]
Did you run the normal install script? And is is shown in the "Plasmasearch" plugin list if you open systemsettings?

**Comment by micmalti** [comment]
Yes and yes. This may be a stupid question but should VS Code be running for the plugin to work?

**Comment by alex1701c** [comment]
No, it should not be running. Stupid question from my side though: Do you have the "Project Manager" plugin installed and configured at least one project?

**Comment by micmalti** [comment]
No, I did not. My bad for failing to realize I had to save projects in order for them to show up in krunner.

**Comment by alex1701c** [comment]
All good, hope it works great for you now 🚀
